pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -58,6 +57,8 @@ import (
 	"knative.dev/pkg/controller"
 	"knative.dev/pkg/logging"
 	pkgreconciler "knative.dev/pkg/reconciler"
+	"knative.dev/pkg/system"
+	_ "knative.dev/pkg/system/testing"
```

Comment:
I think the linter usually asks for a comment on top of `_` import (so that we know why it's there)

Reply:
Ah yeah it doesn't appear to require it for test files. I'll leave a comment anyway.